workflows/requests.go

```diff
@@ -0,0 +1,34 @@
+package workflows
+
+import (
+	"go.temporal.io/sdk/activity"
+	"go.temporal.io/sdk/worker"
+	"go.temporal.io/sdk/workflow"
+)
+
+type WorkflowReference struct {
+	InstanceID string `json:"instance_id"`
+}
+
+// StartRequest is the object describing a Start Workflow request.
+type StartRequest struct {
+	Options           map[string]string `json:"workflow_options"`
+	WorkflowReference WorkflowReference `json:"workflow_reference"`
+	WorkflowName      string            `json:"function_name"`
+	Parameters        interface{}       `json:"parameters"`
+}
+
+// CreateWorkerRequest is the object describing a Create Worker request.
+type CreateWorkerRequest struct {
+	TaskQueue     string         `json:"task_queue"`
```

Comment:
TaskQueue is a Temporal-specific concept and thus shouldn't belong outside of the temporal package. But is the `CreateWorkerRequest` being used anywhere?

Reply:
Follow up question: Is there a design doc for what the workflow interface actually should look like?

At this level there should generally only be things matching the protos and relevant helpers that apply to all components of the workflow type.

Reply:
@RyanLettieri what you put here should match the proto and changes in https://github.com/dapr/dapr/pull/5158